src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/config/RedisConfig.kt

```diff
@@ -6,25 +6,39 @@ import org.springframework.data.redis.core.convert.KeyspaceConfiguration
 import org.springframework.data.redis.repository.configuration.EnableRedisRepositories
 import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.magiclink.MagicLinkConfig
 import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.magiclink.MagicLinkSecret
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.onetimecode.OneTimeCode
+import uk.gov.justice.digital.hmpps.sendlegalmailtoprisonsapi.onetimecode.OneTimeCodeConfig
 
 // we want keyspace notifications, but have to empty the config parameter (default Ex) since elasticache doesn't support
 // changing the config.  If we move off elasticache then need to remove the config parameter and let it use the default.
 @Configuration
 @EnableRedisRepositories(
   enableKeyspaceEvents = RedisKeyValueAdapter.EnableKeyspaceEvents.ON_STARTUP,
   keyspaceNotificationsConfigParameter = "\${standalone-redis.keyspace-notifications:}",
-  keyspaceConfiguration = SecretKeyspaceConfiguration::class
+  keyspaceConfiguration = OneTimeCodeKeyspaceConfiguration::class
```

Comment:
The changes in this class are just to switch to the new configuration (30 minutes rather than 1 hour for the email link). It's not ideal that we can't configure different TTLs for different caches which I believe is due to elasticache (the AWS Redis rip off).  Could have explored further but as we will be throwing one of the caches away I didn't see much point.